pages/common/zmv.md

```diff
@@ -0,0 +1,16 @@
+# zsh
+
+> Move (usually, rename) files matching the regex pattern to corresponding files having names of the form given by destination.
+> More information: <http://zsh.sourceforge.net/Doc/Release/User-Contributions.html>.
+
+- Below renames ‘foo.lis’ to ‘foo.txt’, and so on:
```

Comment:
Maybe this is better:
```suggestion
- Change the extension for all `*.log` files to `.txt`, keeping the same name:
```

Reply:
It's not only an extension rename. We can change other parts of file name so let's keep it generic.

Reply:
But your example can be dedicated for just change extensions.

Reply:
I suppose the example is simple but a bit restrictive.

It lets you define `capturing group` like in regex and use it in the destination file name. That's where I see it shine.

Eg - `zmv '(*) - (*).mp3' '$2 - $1.mp3'` renames `HeartBreaker - Mariah Carey.mp3` to `Mariah Carey - HeartBreaker.mp3`